dashboard/app/models/concerns/text_to_speech.rb

```diff
@@ -100,6 +100,18 @@ def self.tts_upload_to_s3(text, filename)
     end
   end
 
+  def self.sanitize(text)
```

Comment:
Perhaps `sanitize_and_render`? Otherwise, I would expect `sanitize` to accept and return a string, nothing more.